Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
Correct
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-1> in <module>()
     20     received = readln(clientsocket)
     21 while True:
---> 22     num = int(readln(clientsocket))
     23     table = []
     24     seta = []

ValueError: invalid literal for int() with base 10: b'ptm{V3ct0r_m4th_w1th_3m0ji1_i5_fun}'
